packages/web-app/src/routes/index.js

```diff
@@ -26,5 +27,6 @@ router.use(`/${views.invalidAppealDetails}`, handleAppealAlreadyReviewed, invali
 router.use(`/${views.missingOrWrong}`, handleAppealAlreadyReviewed, missingOrWrongAppealDetails);
 router.use(`/${views.document}`, documentsServiceProxy);
 router.use(`/${views.appealAlreadyReviewed}`, appealAlreadyReviewed);
+router.use(`/${views.checkAndConfirm}`, checkAndConfirmDetails);
```

Comment:
might want to consider using `submissions-for-review/check-and-confirm` instead so that the full route is captured since I have a ticket that is very similar to this.